### PR TITLE
Fixes Issue: #719 Added ByteSize.ToFullWords() to return full words instead of abbreviations

### PR DESCRIPTION
**Pull request by Ashera138**
Here is a checklist you should tick through before submitting a pull request: 
 - [x] Implementation is clean
 - [x] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [x] No ReSharper warnings
 - [x] There is proper unit test coverage
 - [x] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [x] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [x] Xml documentation is added/updated for the addition/change
 - [x] Your PR is (re)based on top of the latest commits from the `dev` branch (more info below)
 - [x] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [x] Readme is updated if you change an existing feature or add a new one
 - [] Run either `build.cmd` or `build.ps1` and ensure there are no test failures
Fixes issue #719.

I'm getting a failing test when running build.cmd, but it's unrelated to any code I changed. The picture below shows this:

testfailure

Please let me know if I need to make any changes. Thank you very much! 😄

### CODE REVIEW

**Comment by onovotny** [comment]
This looks good, but my only question is if the words should be upper-case? I think it should be something like 10 megabytes, not 10 Megabytes?

Also, how does it handle singular vs plural? 1 megabyte vs 10 megabytes? It's not as much of an issue with abbreviations, but I think would stick out when expanded to words?

File: readme.md [hunk 1]
```diff
@@ -903,7 +903,7 @@ Also the reverse operation using the `FromMetric` extension.
 ```
 
 ### <a id="bytesize">ByteSize</a>
-Humanizer includes a port of the brilliant [ByteSize](https://github.com/omar/ByteSize) library.
+Humanizer includes a part of the brilliant [ByteSize](https://github.com/omar/ByteSize) library.
```
Comment (hangy) [inline readme.md]: "port" should be correct IMHO

**Comment by hangy** [comment]
> This looks good, but my only question is if the words should be upper-case? I think it should be something like 10 megabytes, not 10 Megabytes?

That should depend on the locale. In English, they would use "1 byte", but in German, we use "1 Byte".

**Comment by onovotny** [comment]
@hangy question is what the default should be? There are already ways to lower case or TitleCase things. Or if there are known rules, then the implementation could do the right thing based on the locale. I don't know what all the rules are for when it should start upper or not. Is there a list somewhere?

**Comment by Ashera138** [comment]
I'm seeing both uppercase and lowercase used online, but lowercase seems to be more common. I can't find any source that defines actual capitalization standards, but pages such as [these](https://techterms.com/help/data_storage_units_of_measurement) tend to show the words in lowercase.

Also, let me know if you would like `part` changed back to `port`.

**Pull request merged**